ROMFS/px4fmu_common/mixers/vtol_tailsitter_duo.main.mix

```diff
@@ -29,11 +29,11 @@ Channel 5 connects to the right (starboard) elevon.
 Channel 6 connects to the left (port) elevon.
 
 M: 2
-O:        7500    7500    0 -10000  10000
+O:       10000   10000    0 -10000  10000
 S: 1 0   10000   10000    0 -10000  10000
 S: 1 1   10000   10000    0 -10000  10000
 
 M: 2
-O:        7500    7500    0 -10000  10000
```

Comment:
https://github.com/PX4/Firmware/commit/c49e576d607f3d27bf318cfbd24bae95e679e153    

 reduce output scaling as workaround for broken output limits ------Have we solved it elsewhere now?

Reply:
i'm also using the mixer like this to increase the control authority. i don't think there's any other way suggested.

Reply:
Hello, I have some questions. Why limit this to 75% in first?